WindowsServerDocs/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server.md

```diff
@@ -139,6 +139,9 @@ Check the WAP configuration by running the Get-WebApplicationProxyConfiguration
 ```PowerShell
 Get-WebApplicationProxyConfiguration
 ```
+> [!NOTE]
```

Comment:
You may want to add a blank line between previous formatting code and the new one:
```suggestion

> [!NOTE]
```
This is mostly a safety & codestyle advice. Even though the current content may work well, it is mostly because of supported compatibility.